lab_Arrays/src/main/java/comper/Europa.java

```diff
@@ -0,0 +1,32 @@
+package comper;
+
+public class Europa  implements Comparable<Europa>{
```

Comment:
could Asia or America or Africa be comparable and have country and capital ? - yes
if so we could define another level of abstraction for example Country that will declare common behaviour and fields and then Auropa America could extend it )) 
Just think about OOP concepts when writing code and creating classes :)